### ISSUE TITLE
Create skeleton for API endpoints

### ISSUE THREAD

**Issue by kanafghan** [kanafghan/fiziq-backend#16]
The scope of this task is only to implement the skeleton for the various API endpoints defined in #6. The logic of each endpoint is going to be a separate task.

**Comment by kanafghan** [comment]
As mentioned in the description of this issue, this issue depends on #6.
